test-packages/test-services-e2e/TripPin/microsoft-o-data-service-sample-trippin-in-memory-models-service/Airlines.d.ts

```diff
@@ -0,0 +1,91 @@
+import { AirlinesRequestBuilder } from './AirlinesRequestBuilder';
```

Comment:
I'm a bit confused.
All the changes in this folder are about adding `.d.ts` files and `.d.ts.map` files, which do exist in the current master branch.